administrator-manual/en/webtop5.rst

```diff
@@ -346,6 +346,57 @@ Sharing Contacts
 In the same way, you can always share your contacts by selecting the directory you want to share -> right click -> "Sharing and permissions".
 Select the recipient user of the share (or Group), and enable permissions for both the folder and the individual items.
 
+Resource calendars
+==================
+
+With impersonal resource calendars it is possible to manage reservations exclusively for company resources such as company cars, meeting rooms and shared equipment in general.
+The reservation of a resource takes place during the creation of a calendar event by adding among the guests the necessary resources for that specific time period, being able to immediately evaluate their real availability based on the other reservations already made.
+
+Creating a new resource
+-----------------------
+
+Before creating a new resource in the WebTop admin panel it is necessary to create a specific user (in the configured user base) which will be dedicated to the resource.
+
+.. note::
+
+   If the newly created user is not used for other purposes, it is advisable to block it and to make his built-in email address only available for internal use
+
+Once the account has been created, you can access the WebTop admin panel to create the new resource and fill in the required fields:
+
+:guilabel:`Name:` name of the resource which must coincide with the user created on the account provider
+
+:guilabel:`Display Name:` description of the resource that will be displayed as a calendar
+
+:guilabel:`Type:` choose the type of resource between Room and Equipment
+
+:guilabel:`Available Resource:` if not selected, the resource will not be available and bookable by the users
```

Comment:
What is this for? I can't understand how it works.
Maybe is it better to add an example?